README.md

```diff
@@ -162,8 +162,42 @@ Via display names (introduced in V3.0RC01 Referable/displayName, it is also poss
 
 Note: It is requested that the idShort of a non-identifiable is unique in its name space (for example in a Submodel or in a SubmodelElementCollection). This is not requested for the display names!
 
-
-
+**[How to reference an ECLASS application or classification class](#id45)** <a id="id45"></a>
+
+Referencing of application and classification classes is no different to referencing the ECLASS property definition.
+Since classification and application classes correspond to a set of properties they can be used to describe the semantics of a complete submodel or of an SubmodelElementCollection.
+
+Please be aware: to be completeley unambiguous it is either necessary to model the complete hierarchy in the submodel or by using a semanticId with more than one IRDI. The keys in the reference are ordered.
```

Comment:
In ECLASS a classification class can only be contained in exactly another classification class. IEC61360 would also allow that a classification class is member of more than one other classification class. So for ECLASS property together with its parent classification class is sufficient for a unique semanticId. ==> change example